src/resources/basis.js

```diff
@@ -227,6 +228,8 @@ let initializing = false;
  * @param {string} [config.glueUrl] - URL of glue script.
  * @param {string} [config.wasmUrl] - URL of the wasm module.
  * @param {string} [config.fallbackUrl] - URL of the fallback script to use when wasm modules aren't supported.
+ * @param {boolean} [config.lazyInit] - Wait for first transcode request before initializing basis
+ * (default is false). Otherwise initialize basis immediately.
```

Comment:
Can we get consistency across the codebase about capitalization of Basis? The master repo has it capitalized:
https://github.com/BinomialLLC/basis_universal

Reply:
oh gosh, yes sorry

Reply:
done